cmd/argocd/commands/project.go

```diff
@@ -84,6 +85,8 @@ func NewProjectCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	command.AddCommand(NewProjectRemoveSourceNamespace(clientOpts))
 	command.AddCommand(NewProjectAddDestinationServiceAccountCommand(clientOpts))
 	command.AddCommand(NewProjectRemoveDestinationServiceAccountCommand(clientOpts))
+	command.AddCommand(NewProjectAddNodeLabelCommand(clientOpts))
```

Comment:
I dont think the configuration should be on a per-project basis. The labels are usually different per clusters, but this sounds more like an admin configuration rather than a project restriction.